ui/app/utils/path-tree.js

```diff
@@ -39,11 +39,14 @@ export default class PathTree {
   /**
    * @param {MutableArray<VariableModel>} variables
    */
-  constructor(variables) {
+  constructor(variables, { delimiter = '/' } = {}) {
```

Comment:
I stopped short of fully generifying this (changing `.files[]` to `.terminal[]` etc.) but did:
- parameterize the delimiter
- add `prefix` as a returned field for ease of use in the UI